src/utils/molecules.js

```diff
@@ -0,0 +1,9 @@
+import { get } from 'lodash-es';
+
+export function has3dCoords(molecule) {
+  const coords = get(molecule, 'cjson.atoms.coords.3d')
+  if (coords === undefined || coords.length == 0)
```

Comment:
Can't we just return `coords !==  undefined && coords.length >  0`

Reply:
Sure!

Reply:
Done.